engine/src/main/java/org/terasology/rendering/nui/layers/mainMenu/savedGames/GameInfo.java

```diff
@@ -17,6 +17,7 @@
 
 import org.terasology.game.GameManifest;
 
+import java.awt.image.BufferedImage;
```

Comment:
If you're in a class and spot

```
/**
 */
```

For class level javadoc (a few lines below this comment, the import is fine) please remove it or better yet, fill it in! :-)

Reply:
ok